src/app/user-times/user-times.component.ts

```diff
@@ -131,13 +136,18 @@ export class UserTimesComponent implements OnInit{
             return;
         }
 
+        if (this.weekLoaded !== undefined){
+            this.weekLoaded.unsubscribe();
+        }
+
+        this.weekChooseDisabled = true;
         this.resetTotals();
         this.resetHours();
         this.currentWeekId = this.currentWeekId + delta;
         const newDate = new Date(this.weeks.get(this.currentWeekId).begin);
         this.tsdayssService.date = new Date(newDate.getUTCFullYear(), newDate.getUTCMonth(), newDate.getUTCDate());
         this.days.clear();
-        this.tsdayssService.getDays(this.email, this.currentWeekId).subscribe(
+        this.weekLoaded = this.tsdayssService.getDays(this.email, this.currentWeekId).subscribe(
```

Comment:
I think you might not have rebased this - i was calling this `daysSubscription` and it's not showing as being changed